synapse/lib/node.py

```diff
@@ -74,7 +74,7 @@ async def iterEdgesN2(self, verb=None):
         async for edge in self.snap.iterNodeEdgesN2(self.buid, verb=verb):
             yield edge
 
-    async def storm(self, text, opts=None, user=None, path=None):
+    async def storm(self, runt, text, opts=None, path=None):
```

Comment:
some of these api changes feel a bit odd. since custom code can just open a cortex, get a snap and lift a node, these node-specific funcs become a function of having a runtime available as well?